scheduler/src/cook/mesos/api.clj

```diff
@@ -198,6 +198,7 @@
               :status s/Str
               :submit-time PosInt
               :retries-remaining NonNegInt
+              (s/optional-key :successful) s/Bool
```

Comment:
@wyegelwel What would you think about simply adding to the possible values returned by the existing "status" field?  If we have both "status" and e.g. "descriptive_state" that makes the API hard to explain.   There isn't necessarily a mandate to keep the Datomic schema resembling the API schema, when we actually do want to convey different information in each context.